<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PySide Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seven">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_one">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zero">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_four">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_two">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eight">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_five">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_three">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ten">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_six">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiple">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>52</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>17</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
